lib/napa/active_record_extensions/notifications_subscriber.rb

```diff
@@ -1,17 +1,17 @@
 if defined?(ActiveRecord)
-  ActiveSupport::Notifications.subscribe 'sql.active_record' do |name, start, finish, id, payload|
+  ActiveSupport::Notifications.subscribe 'sql.active_record' do |_name, start, finish, _id, payload|
     if payload[:sql].downcase.match(/(select|update|insert|delete)(.+)/i)
       table, action = Napa::ActiveRecordStats.extract_sql_content(payload[:sql])
     end
 
     if table
       Napa::Stats.emitter.timing(
-        "sql.query_time",
```

Comment:
you no like double quotes? Any reason in particular for this change?

[ref](https://viget.com/extend/just-use-double-quoted-ruby-strings#tldr)